contrib/linearize/linearize-hashes.py

```diff
@@ -8,10 +8,7 @@
 #
 
 from __future__ import print_function
-try: # Python 3
-    import http.client as httplib
-except ImportError: # Python 2
-    import httplib
+import http.client as httplib
```

Comment:
Keeping this alias seems strange. I think better to just import the one symbol that is being used:

```from http.client import HttpConnection```

(and then change `httplib.HTTPConnection` to `HTTPConnection` lower down)